clusters/app.ci/quay/group-quay-admins.yaml

```diff
@@ -0,0 +1,6 @@
+kind: Group
+apiVersion: v1
+metadata:
+  name: quay-admins
+users:
+- dmage
```

Comment:
@flavianmissi @ricardomaraschini if you want to have access to this namespace on app.ci, please give me your usernames

Reply:
fmissi